examples/common/golioth_basics.c

```diff
@@ -85,16 +85,17 @@ static enum golioth_rpc_status on_multiply(zcbor_state_t *request_params_array,
 
```

Comment:
This commit message needs to be updated for the new member names: `status_class` used instead of `class` (which is a protected name in CPP).

This commit should also have a breaking change entry in the CHANGELOG.

Reply:
We should use `code_class` and `code_detail` to align with the CoAP spec:

```
Code:  8-bit unsigned integer, split into a 3-bit class (most
      significant bits) and a 5-bit detail (least significant bits),
      documented as "c.dd" where "c" is a digit from 0 to 7 for the
      3-bit subfield and "dd" are two digits from 00 to 31 for the 5-bit
      subfield.  The class can indicate a request (0), a success
      response (2), a client error response (4), or a server error
      response (5).  (All other class values are reserved.)  As a
      special case, Code 0.00 indicates an Empty message.  In case of a
      request, the Code field indicates the Request Method; in case of a
      response, a Response Code.  Possible values are maintained in the
      CoAP Code Registries (Section 12.1).  The semantics of requests
      and responses are defined in Section 5.
```

`status` has no canonical meaning in this context.